pkg/mock/fileUtils.go

```diff
@@ -256,6 +256,43 @@ func (f *FilesMock) FileRemove(path string) error {
 	return nil
 }
 
+// RemoveAll deletes the association of the given path with any content and records the removal of the path.
+// If the path has not been registered before, it returns an error.
+func (f *FilesMock) RemoveAll(path string) error {
+	if f.files == nil {
```

Comment:
any reason why we cannot delegate to the default `RemoveAll()` function?

Reply:
This is an addition to the FilesMock, it cannot delegate to the default RemoveAll() function, it's supposed to mock it.

BTW, thanks for this addition! I've had a TODO in the WhiteSource step for this exact functionality.